OutOfSchool/OutOfSchool.WebApi/Controllers/V1/ChatWorkshopController.cs

```diff
@@ -175,6 +164,36 @@ public Task<IActionResult> GetParentsRoomsAsync()
     public Task<IActionResult> GetProvidersRoomsAsync()
         => this.GetUsersRoomsAsync(providerId => roomService.GetByProviderIdAsync(providerId));
 
+    /// <summary>
+    /// Get a chat room for current parent and workshopId.
+    /// </summary>
+    /// <param name="workshopId">WorkShop's Id.</param>
+    /// <returns>ChatRoom that was found.</returns>
+    [HttpGet("parent/chatroomforworkshop/{workshopId}")]
+    [Authorize(Roles = "parent")]
```

Comment:
we're using permissions, so maybe use one of the permissions instead of role, like ParentRead/ProviderRead if applicable?

Reply:
The user, who call this endpiont, must be "parent" in our logic.